StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/CommonMemberData.cs

```diff
@@ -137,5 +137,13 @@ public static IEnumerable<object[]> TypeKeywordsWhichSupportPrimaryConstructors
                 }
             }
         }
+
+        public static IEnumerable<object[]> ReferenceTypeKeywordsWhichSupportPrimaryConstructors
```

Comment:
💡 Consider rewriting this the other way around so `TypeKeywordsWhichSupportPrimaryConstructors` calls `ReferenceTypeKeywordsWhichSupportPrimaryConstructors`.